iml-gui/crate/src/generated/css_classes.rs

```diff
@@ -3927,6 +3927,381 @@ pub struct CssClasses<'a> {
     */
     pub border_l: &'a str,
 
+    /**
```

Comment:
Just making sure all of these classes are necessary

Reply:
We don't use most of them, but a production build prunes out any unused classes anyway.